index.html

```diff
@@ -74,6 +70,10 @@
     </div>
   </footer>
 
+  <div id="add-project-form">
+    <iframe src="https://docs.google.com/forms/d/1CiNYA3gzHPd4HYrgQuQB2KM0TDpsbn6jbMcl-0_OjM4/viewform?embedded=true" width="100%" height="100%" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+  </div>
```

Comment:
didn't change this part much except

1. moved this block down
2. changed it from `<section>` to `<div>`